src/oatpp/core/concurrency/Thread.cpp

```diff
@@ -23,9 +23,10 @@
  ***************************************************************************/
 
 #include "Thread.hpp"
+#include "oatpp/core/base/Environment.hpp"
 
 #if defined(_GNU_SOURCE)
-  #include <pthread.h>
+#include <thread>
```

Comment:
I think this guy should be  `<pthread.h>` since we are using `pthread_setaffinity_np` method